float_api/float_api.py

```diff
@@ -134,15 +134,15 @@ def _get_all_pages(self, path, error_object, params = {}):
       list_to_return += l
 
       # Exit loop if we are on the last page of results
-      if int(r.headers['X-Pagination-Current-Page']) >= int(r.headers['X-Pagination-Page-Count']):
+      if int(r.headers.get('X-Pagination-Current-Page',1)) >= int(r.headers.get('X-Pagination-Page-Count',1)):
         break
 
       # Next page
       params['page'] += 1
 
-
-    # All records must be in the list to return
-    assert int(r.headers['X-Pagination-Total-Count']) == len(list_to_return), "Get all returns all records"
+    if int(r.headers.get('X-Pagination-Page-Count',1)) > 1:
+      # All records must be in the list to return
+      assert int(r.headers['X-Pagination-Total-Count']) == len(list_to_return), "Get all returns all records"
```

Comment:
Do we get responses with a _X-Pagination-Total-Count_ but not a _X-Pagination-Page-Count_?

Reply:
This is the same change proposed in #26 - As without it, the tests fail.

Reply:
Let's get #26 merged in before this.